app/models/related_instruction.rb

```diff
@@ -8,4 +10,16 @@ class RelatedInstruction < ApplicationRecord
 
   delegate :title, :description, :code, :hours, to: :default_course, prefix: true
   delegate :title, to: :organization, prefix: true
+
+  def hours_in_human_format
```

Comment:
Unfortunately, we can't call helper methods in the word template, so adding this method to the model seemed like the best alternative for me. I'm exploring adding them to a decorator, but I'd like not to block this feature because of that. Patrick has mentioned that someone on the team could test out the templates before they are available, so we can merge this, and improve this code while people run their tests.

Reply:
Should we extract the method into a concern?

Reply:
Yes, thanks for suggesting - I will look into extracting these into a decorator or a concern